docs/quickstart.mdx

```diff
@@ -160,13 +312,6 @@ deeper, please check out the [develop] chapter and join the discussion on the
 
 Best of luck on your future forays into confidentiality!
 
-:::info Example: Hardhat
-
-Visit the Sapphire ParaTime repository to download the [Hardhat][hardhat-example]
```

Comment:
This should still be part of the repo, so we have the Quickstart tested. Perhaps under `examples/hardhat`? Ideally, we should also include snippets directly from the code (e.g. using blocks with #region and #endregion) instead of pasting it into markdown.

Reply:
This can be done in a followup PR.

Reply:
Makes sense. Will add `examples/hardhat`. https://github.com/oasisprotocol/sapphire-paratime/pull/524